resources/views/users/donated-items/donated_item.blade.php

```diff
@@ -10,56 +10,49 @@
-                                'r2' => route('item.destroy', $donationItem->id)
-                            ])
-                            @slot('body')
-                            <p class="h5 text-center">Are you sure you want to delete this item?</p>
-                            <p class="h6 text-center" style="color: #9EA6A6;">
-                            This item will be permanently deleted from all pages! <br>
-                            This cannot be undone.
-                            </p>
-                            @endslot
-                            @endcomponent
-                            @endif 
-                        </div> 
+                <div class="d-flex justify-content-center align-items-center h1 mt-3">
+                    <i class="fa-solid fa-circle-info ps-1"></i>
+                    <p class="mb-0 ms-4">Item's information</p>
+                @if(Auth::user()->id !== $donationItem->user->id)
+                    @if($donationItem->isFavorited())
+                        <form action="{{ route('favorite.destroy', ['donationItem_id' => $donationItem->id]) }}" method="post" class="mb-0">
+                        @csrf
+                        @method('DELETE')
+                        <button type="submit" class="new-bookmark ms-3"><i class="fa-solid fa-bookmark text-dark"></i></button>
+                        </form>
+                    @else
+                        <form action="{{ route('favorite.store', ['donationItem_id' => $donationItem->id]) }}" method="post" class="mb-0">
+                        @csrf
+                        <button type="submit" class="new-bookmark ms-3"><i class="fa-regular fa-bookmark"></i></button>
+                        </form>
+                    @endif
+                @endif
+                @if(Auth::user()->id === $donationItem->user->id)
+                     <a href="{{ route('donated.item.edit', $donationItem->id) }}" class="btn">
+                        <i class="fa-solid fa-pen gray" ></i>
+                    </a>
+                    <div class="tooltip-container">
+                        <button class="btn" data-bs-toggle="modal" data-bs-target="#delete-modal">
+                            <i class="fa fa-trash-can"></i>
+                            <span class="h5 mb-2 text-danger add">Don't delete this item during dealing.</span>
+                        </button>
                     </div>
-                </div>
-
+                    {{-- Component Delete Modal --}}
+                    @component('users.components.deletemodal', [
+                         'id' => 'delete-modal',
+                         'title' => 'Delete Item',
+                         'r2' => route('item.destroy', $donationItem->id)
+                        ])
+                    @slot('body')
+                    <p class="h5 text-center">Are you sure you want to delete this item?</p>
+                    <p class="h6 text-center" style="color: red;">
```

Comment:
style color red == text-danger in bootstrap. Better to use class rather than inline style